sdv/single_table/copulagan.py

```diff
@@ -144,4 +144,43 @@ def __init__(self, metadata, enforce_min_max_values=True, enforce_rounding=True,
             field: GaussianCopulaSynthesizer._validate_distribution(distribution)
             for field, distribution in (numerical_distributions or {}).items()
         }
+
+    def _create_gaussian_normalizer_config(self, processed_data):
+        columns = self.metadata._columns
+        transformers = {}
+        sdtypes = {}
+        for column in processed_data.columns:
+            column_name = column.replace('.value', '')
+            sdtype = columns.get(column_name, {}).get('sdtype')
+            if column_name in columns and sdtype != 'categorical':
```

Comment:
should we be doing the same with boolean columns? Perhaps `CopulaGANSynthesizer` should mark booleans as categorical as well.

Reply:
Yes I forgot to ask about this, I left it as the legacy code, but I think that the intention was to use booleans as well. Also, should booleans arrive as `True - False` rather than numerical `0. - 1.` ? I think that was the intention of the legacy code but that was not really achived.

Reply:
I think it's fine either way. You can pass as `True` and `False`